pages/introduction/privacy-policy/README.md

```diff
@@ -0,0 +1,32 @@
+---
+navigation_title: "Privacy Policy"
+position: 5
+changed: "2020-04-30"
+---
+
+# Privacy Policy
+
+In order for us to provide you the best possible experience on our guide, we need to collect and process certain information.
+
+## Usage data
+
+When you visit our site, we will store amongst other things:
+
+- The website from which you visited us from
+- The parts of our site you visit
+- The date and duration of your visit
+- Your anonymised IP address
+- Information from the device (device type, operating system, screen resolution, language, country you are located in, and web browser type) you used during your visit
+
+We process this usage data in [Matomo Analytics](https://matomo.org/) for statistical purposes, to improve our site.
+
+## Cookies
+
+We use cookies (small data files transferred onto computers or devices by sites) for record-keeping purposes and to enhance functionality on our site. You may deactiva
+
+## Opt-out of website tracking
+
+<iframe
+        style="border: 0; height: 300px; width: 100%;"
+        src="https://adg-analytics.nothing.ch/index.php?module=CoreAdminHome&action=optOut&language=de&fontColor=272727&fontSize=20px&fontFamily=sans-serif"
```

Comment:
Last but not least: is it enough to place this information and opt-out option deeply hidden in the website? Shouldn't it be advertised on the homepage or something?

At least we should place a link in the footer, I guess:

![image](https://user-images.githubusercontent.com/1814983/85556137-d9f5d280-b626-11ea-8750-7ba8bf83e444.png)

Reply:
I jut added this myself now.

Reply:
Yes, we agreed that it's enough.